lib/publiccloud/instance.pm

```diff
@@ -639,6 +639,7 @@ sub store_boottime_db() {
     my $token = get_var('_SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN');
 
     return unless ($results && $data_push && $url);
+    return if is_openstack;
```

Comment:
this would be wrong place to skip . https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/provider.pm#L337  here would be better place . because currently you would spent time on collecting data which you actually don't need

Reply:
ah no, I was mislead. I will fix this then.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18801